{{cookiecutter.repo_name}}/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,30 +6,13 @@ The PR submission process is described in [more detail here.](http://edx.readthe
 - [ ] Manually test right-to-left languages and i18n
   of the changes.
 
-### DevOps assistance
-If it's needed, or even if you suspect it might be needed, do any/all these
-things as early as possible. Don't surprise devops right before merging!
-For example, touching envs/aws.py or or envs/common.py are dead giveaways
-that you'll need assistance when pushing your changes out.
-
-Things that require devops attention include, but are not limited to:
-- configuration changes
-- feature flags
-- migrations
-
-Ways to get devops attention, pick-and-choose whichever is most appropriate for your specific PR:
-- TODO: Tag devops on your PR (user `edx/devops`), either for review (more specific to devops = better) or just a general FYI
-- TODO: File a support ticket
-
-FYI: Tag anyone who might be interested in this PR here.
-
-### Documentation considerations
-- Are there user visible changes? If so, describe which users. Consider learners, course teams,
-  researchers, installers/maintainers, and developers.
-- Your commit messages will be the first draft used for release notes. Please include a meaningful
-  subject line, and a detailed body if more explanation is needed.
+## Non-testing Checklist
+- TODO: Tag DevOps on your PR (user `edx/devops`), either for review
+  (more specific to devops = better) or just a general FYI.
+- TODO: Consider any documentation your change might need.
```

Comment:
That's already in the `CONTRIBUTING.md`. Do you think it needs to be explicitly called out in the template?